src/containers/AccessPointDetails/components/OS/components/Timer/index.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import moment from 'moment';
+
+const Timer = () => {
+  const [date, setDate] = useState(new Date().toLocaleString());
+
+  useEffect(() => {
+    const secTimer = setInterval(() => {
+      setDate(new Date().toLocaleString());
+    }, 1000);
+
+    return () => {
+      clearInterval(secTimer);
+    };
+  }, []);
```

Comment:
change this to:
`useEffect(() => {
    setTimeout(() => setDate(date.add(1, 'second')), 1000);
  }, [date]);`